lib/realtime/data/vehicle_position.ex

```diff
@@ -0,0 +1,34 @@
+defmodule Realtime.Data.VehiclePosition do
+  @moduledoc """
+  The subset of the VehiclePosition data in the enhanced_json feed that we use.
+  """
+  use Ecto.Schema
+
+  @type t :: %__MODULE__{
+          route_id: Realtime.Data.route_id() | nil,
+          direction: integer(),
+          label: String.t(),
+          position: Util.Position.t() | nil,
+          heading: float() | nil,
+          station_id: String.t() | nil,
+          current_status: :INCOMING_AT | :STOPPED_AT | :IN_TRANSIT_TO,
+          timestamp: DateTime.t() | nil,
+          # GTFS-RT id, e.g. "R-54826B16".
+          vehicle_id: String.t() | nil,
+          trip: nil
```

Comment:
note: `trip: nil` in the typespec doesn't make sense in the long term, as it means "trip is present, but it always nil". In Glides this is how the server-side trip match is sent to the client (it's a `virtual` field, so it isn't persisted), but it does make sense that we'll ultimately be putting OCS trip data here. So we can leave it 👍